src/popup/components/MergeRequest.tsx

```diff
@@ -57,7 +55,10 @@ export const MergeRequest = (props: Props) => {
                         </Label>
                     </div>
                 </Box>
-                <Box className={'avatarsList'}>{listAvatars}</Box>
+                <Box className={'avatarsList'}>
+                    {avatars}
+                    {mr.assignees.length > 3 ? <Octicon icon={Plus} /> : ''}
```

Comment:
Consider adding some spacing in the display, the "+" is not rendering well sometimes. (also we need to make sure it doesn't break the design when the title of the MR is super long).

Reply:
we should truncate very long MR names, what do you think_

Reply:
It's already the case in the `style.css` file in fact
There is a misunderstanding, i meant to reduce a bit the max-width of the title if you add a "+" symbol in the avatars list.

Reply:
I increased avatars max size and reduced title's